### ISSUE TITLE
Incorrect default specified for time unit of measurement in scheduler.py

### ISSUE THREAD

**Issue by jfrankle** [mosaicml/composer#511]
In `composer/composer/optim/scheduler.py`:

```
_interval_doc = 'frequency of step() calls, either "batch" or "epoch". Default: "epoch"'
INTERVAL_MAP = {    'batch': 'batch',    'batches': 'batch',    'epoch': 'epoch',    'epochs': 'epoch',    'step': 'batch',    'steps': 'batch'   }
@dataclass
class MultiStepLRHparams(SchedulerHparams):
    """Hyperparameters for the `MultiStepLR <https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.MultiSte
    pLR.html#torch.optim.lr_scheduler.MultiStepLR>`_ scheduler."""

    milestones: List[str] = hp.required(doc='List of milestone time strings')
    gamma: float = hp.optional(default=0.1, doc='multiplicative factor of decay')
    verbose: bool = hp.optional(default=False, doc='prints message to stdout')
    interval: str = hp.optional(default='step', doc=_interval_doc)

    scheduler_object = torch.optim.lr_scheduler.MultiStepLR
```

The default interval in `MultiStepLRHparams` seems to be step which maps to batch, but the doc says that the default is `epoch`

**Comment by jbloxham** [comment]
The scheduler PR removes the `interval` param altogether, so this is inapplicable.

**Comment by ravi-mosaicml** [comment]
Sounds good; closing the issue as #463 is merged.
